src/components/ui/FiltersPart.tsx

```
@@ -27,44 +31,97 @@ const tabletModels = ["iPad Pro", "iPad Air", "Samsung Tab"];
-    {label}<span className="w-3 h-3 text-blue-600">×</span>
+const Badge = ({
+  label,
+  onRemove,
+}: {
+  label: string;
+  onRemove: () => void;
+}) => (
+  <span
+    className="inline-flex cursor-pointer items-center gap-1 rounded-full bg-blue-100 px-2 py-1 text-xs font-medium text-blue-800 hover:bg-blue-200"
+    onClick={onRemove}
+  >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)